source/get-started/next-steps.txt

```diff
@@ -0,0 +1,15 @@
+.. _pymongo-get-started-next-steps:
+
+==========
+Next Steps
+==========
+
+Congratulations on completing the quick start tutorial!
```

Comment:
S: This sounds a little clunky. Can we just say "tutorial"? or "Get Started guide"?

```suggestion
Congratulations on completing the tutorial!
```

Reply:
Yeah. Now it repeats 'tutorial'. What about the following? It also shortens it into one paragraph, which you may or may not like...

Congratulations on completing the tutorial! You created a Python application that connects to a MongoDB deployment hosted on MongoDB Atlas and retrieves a document that matches a query.